src/frontend/src/flows/manage/index.ts

```diff
@@ -24,20 +24,25 @@ import { mainWindow } from "../../components/mainWindow";
 
 /* Template for the authbox when authenticating to II */
 export const authnTemplateManage = (): AuthnTemplates => {
-  const wrap = (slot: string): TemplateResult => html`
-    <div class="t-centered l-stack">${slot}</div>
+  const wrap = (slot: string, title: string): TemplateResult => html`
```

Comment:
```suggestion
  const wrap = ({slot, title}: {slot: string, title: string}): TemplateResult => html`
```
We've started using named args whenever possible